src/main/perf/LineFileDocs.java

```diff
@@ -471,6 +486,29 @@ static Document cloneDoc(Document doc1) {
     return doc2;
   }
 
+  private String getRandomWord(String body) {
+    Random random = ThreadLocalRandom.current();
+    String[] bodyArr = body.split(" ");
+    if (bodyArr.length == 0) {
+      return "EMPTY_WORD";
+    }
+    int counter = 0;
+    String randomWord = "";
+    // tries 5 times to get a non-empty word
```

Comment:
Oh no, do we have really crappy documents that have many empty terms or something?

Reply:
Yah I was getting empty terms that caused indexing errors. I didn't look into it too much since I figured it wouldn't affect the benchmark since we could just treat this like another random term.